cif_pow.dic

```diff
@@ -8672,8 +8567,8 @@ save_PD_PHASE
     structural analysis is performed, the structural results will be placed in
     different data blocks, using CIF entries from the core CIF dictionary.
 
-    The _pd_phase_block.id or _pd_phase.id entry points to the CIF block with
-    structural parameters for each crystalline phase.
+    The _pd_phase.id entry points to the CIF block with structural parameters
+    for each crystalline phase.
```

Comment:
`_pd_phase.id` doesn't really "point" to anything, it just identifies a phase.

Reply:
I've changed some words and moved that sentence to the actual _pd_phase.id description.

Reply:
How about "_pd_phase.id can be used to associate a block with a particular crystalline phase."